packages/trader/src/AppV2/Components/RiskManagementItem/risk-management-item.tsx

```diff
@@ -186,7 +186,9 @@ const RiskManagementItem = observer(
                             )}
                         </ActionSheet.Content>
                         <ActionSheet.Footer
-                            isPrimaryButtonDisabled={!!errorMessage || finalValue == stepperValue}
+                            isPrimaryButtonDisabled={
+                                !!errorMessage || finalValue == stepperValue || stepperValue === '' || stepperValue == 0
```

Comment:
Will it work if we check falsy values for stepperValue by one check?
stepperValue === '' || stepperValue == 0 ---> !stepperValue